components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2086,6 +2086,24 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         }
     }
 
+    async createOrUpdateStripeCustomerForUser(ctx: TraceContext, currency: string): Promise<void> {
```

Comment:
:cloud: Originally I thought about merging these two methods on `attributionId`. But I think it makes sense to have them separate as they have different semantics around access rights etc.